docs/releases/changelog.md

```diff
@@ -27,19 +46,125 @@ supported with NVCC + MSVC. It also adds documentation.
 
 ## libcu++ 1.2.0 (CUDA Toolkit 11.1)
 
-TODO
+libcu++ 1.2.0 adds `<cuda/pipeline>`/`cuda::pipeline`, a facility for
+  coordinating `cuda::memcpy_async` operations.
+This release introduces ABI version 3, which is now the default.
+
+Supported ABI versions: 3 (default) and 2.
 
-## libcu++ 1.1.1 (CUDA Toolkit 11.0)
+Included in: CUDA 11.1.
+
+### ABI Breaking Changes
+
+- ABI version 3 has been introduced and is now the default.
+  A new ABI version was necessary to improve the performance of
+    `cuda::[std::]barrier` by changing its alignment.
+  Users may define `_LIBCUDACXX_CUDA_ABI_VERSION=2` before including any libcu++
+    or CUDA headres to use ABI version 2, which was the default for the 1.1.0 /
+    CUDA 11.0 release.
+  Both ABI version 3 and ABI version 2 will be supported until the next major
+    CUDA release.
+
+### New Features
+
+- `<cuda/pipeline>`: `cuda::pipeline`, a facility for coordinating
+    `cuda::memcpy_async` operations.
+- `<cuda/std/version>`: API version macros `_LIBCUDACXX_CUDA_API_VERSION`,
+    `_LIBCUDACXX_CUDA_API_VERSION_MAJOR`, `_LIBCUDACXX_CUDA_API_VERSION_MINOR`,
+    `_LIBCUDACXX_CUDA_API_VERSION_PATCH`.
+- ABI version switching: users can define `_LIBCUDACXX_CUDA_ABI_VERSION`
+    to request a particular supported ABI version.
+  `_LIBCUDACXX_CUDA_ABI_VERSION_LATEST` is set to the latest ABI version, which
+    is always the default.
+
+### Other Enhancements
 
-TODO
+- `<cuda/latch>`/`<cuda/semaphore>`: `<cuda/*>` headers added for `cuda::latch`,
+    `cuda::counting_semaphore`, and `cuda::binary_semaphore`.
+   These features were available in prior releases, but you had to include
+      `<cuda/std/latch>` and `<cuda/std/semaphore>` to access them.
+- NVCC + GCC 10 support.
+- NVCC + Clang 10 support.
 
-## libcu++ 1.1.0 (CUDA Toolkit 11.0 Early Access)
+## libcu++ 1.1.0 (CUDA Toolkit 11.0)
 
-TODO
+libcu++ 1.1.0 introduces the world's first implementation of the
+  [Standard C++20 synchronization library](https://wg21.link/P1135):
+  `<cuda/[std/]barrier>`, `<cuda/std/latch>`, `<cuda/std/semaphore>`,
+  `cuda::[std::]atomic_flag::test`, `cuda::[std::]atomic::wait`, and
+  `cuda::[std::]atomic::notify*`.
+An extension for managing asynchronous local copies, `cuda::memcpy_async` is
+  introduced as well.
+It also adds `<cuda/std/chrono>`, `<cuda/std/ratio>`, and most of
+  `<cuda/std/functional>`.
+
+### ABI Breaking Changes
+
+- ABI version 2 has been introduced and is now the default.
+  A new ABI version was introduced because it is our policy to do so in every
+    major CUDA toolkit release.
+  ABI version 1 is no longer supported.
+
+### Breaking Changes
```

Comment:
@griwes @ogiroux did we break anything else in 1.1.0 / 11.0?

Reply:
I don't think the one listed below is really a break - it never really worked in the first place.

Reply:
I'm going to leave it in; we didn't previously say it was unsupported.